src/test/resources/goal-output/commons-collections4-4.1/org.apache.commons.collections4.CollectionUtils_goal.json

```diff
@@ -1271,7 +1271,7 @@
     ],
     "returnTag": {
       "comment": "the last element in the collection, or null if either collection or closure is null",
-      "condition": "args[0]==null || args[1]==null",
+      "condition": "args[0]==null || args[1]==null ? null",
```

Comment:
Another example, this should be: `args[0]==null || args[1]==null ? result == null`